src/lib/types/openapiDoc.ts

```diff
@@ -3245,6 +3245,21 @@ const document = {
                           description:
                             'The description of your project. This is used in the page meta description and is seen by search engines and sites like Facebook.',
                         },
+                        git: {
```

Comment:
Temporarily adding this in manually instead of generating the doc again - will do that once https://github.com/readmeio/readme/pull/13869 is good and merged

Reply:
See https://github.com/readmeio/rdme/pull/1195